tests/test_incident.py

```diff
@@ -117,3 +126,53 @@ def test_inc_revisions(mock_good):
     assert incident.revisions_with_fallback("x86_64", "12-SP5")
     assert not incident.revisions_with_fallback("aarch64", "12")
     assert not incident.revisions_with_fallback("aarch64", "12-SP5")
+
+
+class MockResponse:
+    # TODO: collect all instances where the same pattern is used and refactor,
+    # see openSUSE/qem-bot/issues/161
```

Comment:
Don't add TODO. Better leave it out and keep in your local git repository, either in `git stash` or a temporary "WIP"-commit. We have the issue as reference after all which can suffice.

Reply:
Nope, I'll keep the to-do for the sanity of whoever has to work on this in the future. grepping for `TODO` + support in modern-day editors makes it easier and less prone to be forgotten, should somebody work on it in the future.

If you _absolutely_ don't want to have the todo, do a commit to the branch

Reply:
TODOs in code don't work. Everybody starting to apply changes to the code comes from a certain motivation and they want to get something done. In my experience it is hardly the case that somebody is on it developing a certain feature, stumbling about a TODO and saying "oh, I wanted to get $foo done. I will stop that for now and fix that issue before I continue". If we find other contributors to this repository to convince us that TODO's in code are good then I will accept :)

Reply:
@okurz I'm not having the discussion about TODO's. If you don't want TODOs make an RFC for all of the projects, and we can bring up the same discussion from https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/12972#issuecomment-1073722468

You're subjecting everybody to empirical experience, while in reality [they are quite used](https://github.com/search?q=repo%3Atorvalds%2Flinux+TODO%3A&type=commits) and [helpful](https://github.com/torvalds/linux/commit/2ef08b98025bd09b74f68d1801995b0b068afbe7) overall.

So solve that one first, and then ping me and I'll add the comment :).

Reply:
We don't need an RFC when common reviewers agree on a style and enforce that by reviewing and approving. If there are multiple other project maintainers that tell me that we commonly add TODOs and it's a good thing then I will stand corrected and accept your code as is. So for sure I don't want to just make that my personal opinion.

Reply:
Can't we just replace the TODO by a link to a progress ticket describing the problem? That can assure someone will solve that eventually. We do not have any process in place which involves greping code for TODOs when we are deciding what to do next.